### ISSUE TITLE
Hyperparams examples

### ISSUE THREAD

**Issue by lingz** [inikdom/rnn-speech#12]
It would be cool if you provided a baseline hyperparams so we know what reasonable estimates we could start training with.

**Comment by lingz** [comment]
Just out of curiosity, with the models you have so far, what are your CTC loss, and accuracy looking like (just rough estimate)?

**Comment by inikdom** [comment]
I haven't spent much time on training it (yet), I had one model where ctc loss went down to about 400, and I haven't built in a mechanism to track accuracy yet. I also expect to be able to get significantly better ctc loss results over the next few days. I'm not sure accuracy is they right metric to use here, I want to do my research into that. I think loss is sufficient given the amount of classification labels.

**Comment by lingz** [comment]
Hmm, I managed to get CTC loss to under 200, but it was still at that point giving essentially useless labels (almost all blanks).

**Comment by inikdom** [comment]
Ok, keep in mind for the acoustic model there probably will be a lot of blanks, and the output might look like gibberish, but the language model should clean that up. Of course, I'm not saying that is the case here. 200 might not be a minimized enough loss, I will hopefully be able to get more information by Monday, by allowing some models to train over the next few days.

**Comment by lingz** [comment]
Okay let me know if you see any good results with just the acoustic model.

**Comment by inikdom** [comment]
Hey, just leaving an update here. I've been letting models train over night on a Titan X. It's slow going. I will probably find a decent model over the next week or two, the limiting factors being my time and computational complexity. Will provide another update when I can.

**Comment by AMairesse** [comment]
Hi Ling,
There is some pre-trained acoustic models available now and I've put some comments in the config.ini file to give a recommended value for each hyperparameter.
Does this answer to your bug report ? May I close it ?
Thanks.

**Comment by lingz** [comment]
Yup that's great thanks.
